### PR TITLE
[MM-61099] Fix errcheck issues in server/channels/app/brand.go

### PR DESCRIPTION
**Pull request by Dishika18**
#### Summary
Fixed the errcheck issues.

#### Ticket Link
Fixes: #28779
Jira: [MM-61099](https://mattermost.atlassian.net/browse/MM-61099)

#### Release Note

```release-note
NONE
```

[MM-61099]: https://mattermost.atlassian.net/browse/MM-61099?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hanzei — CHANGES_REQUESTED** [review]
The tests fails with

```
{"timestamp":"2024-11-08 15:23:46.605 Z","level":"error","msg":"brand.save_brand_image.move_image.app_error","caller":"web/context.go:124","path":"/api/v4/brand/image","request_id":"n8ze4ik1kfno3gj8hs7zd7f4uy","ip_addr":"127.0.0.1","user_id":"kfuwfne9zjgb3e331b6hhu9oec","method":"POST","err_where":"SaveBrandImage","http_code":500,"error":"SaveBrandImage: brand.save_brand_image.move_image.app_error, MoveFile: Unable to move file., unable to move the file to brand/2024-11-08T15:23:46.png to the destination directory: rename data/brand/image.png data/brand/2024-11-08T15:23:46.png: no such file or directory"}
{"timestamp":"2024-11-08 15:23:46.605 Z","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:185","method":"POST","url":"/api/v4/brand/image","request_id":"n8ze4ik1kfno3gj8hs7zd7f4uy","user_id":"kfuwfne9zjgb3e331b6hhu9oec","status_code":"500"}
    brand_test.go:75: 
        	Error Trace:	/mattermost/server/channels/api4/brand_test.go:75
        	Error:      	Received unexpected error:
        	            	brand.save_brand_image.move_image.app_error
        	Test:       	TestDeleteBrandImage
```

Can you please check if the tests need to get updated or if the PR introduces a bug? Thanks!

**Comment by Dishika18** [comment]
I have fixed the issue "unable to move the file". Please let me know if it's still showing the error.

**Comment by hanzei** [comment]
@Dishika18  CI fails with the same issue :confused:

**Comment by Dishika18** [comment]
@hanzei Actually, I'm facing an issue I can't figure out. Earlier, the file was undefined, so I handled the error and defined it using `make i18n-extract`. But the issue is still there, and I'm not sure what went wrong. Could you help me out?

**Comment by hanzei** [comment]
Can you please merge `master` into your branch and run `make i18n-extract` again? If that doesn't work, can you please share the error message you are seeing?

**Comment by Dishika18** [comment]
@hanzei I sincerely apologize for the delay. Could you kindly review the PR again?

**Review by hanzei — CHANGES_REQUESTED** [review]

File: server/channels/app/brand.go [hunk 1]
```diff
@@ -45,7 +45,10 @@ func (a *App) SaveBrandImage(rctx request.CTX, imageData *multipart.FileHeader)
 	}
 
 	t := time.Now()
-	a.MoveFile(BrandFilePath+BrandFileName, BrandFilePath+t.Format("2006-01-02T15:04:05")+".png")
+
+	if err := a.MoveFile(BrandFilePath+BrandFileName, BrandFilePath+t.Format("2006-01-02T15:04:05")+".png"); err != nil {
+		return model.NewAppError("SaveBrandImage", "brand.save_brand_image.move_image.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
```
[the remark this answers is not in the document]
Comment (Dishika18) [reply server/channels/app/brand.go:50]: @hanzei I’ve been looking into it, but I’m not able to figure out the problem.

**Comment by Dishika18** [comment]
@hanzei I think the issue might be with this branch, so I'll close this PR and open a new one soon.

**Pull request closed without merging**